### PR TITLE
Update semantic_slots.md

### LINKED ISSUE #6954: Broken Link - Fabric semantic colors documentation
Speaking of permanent semantic color slots - The documentation on Fabric semantic color slots is broken and link goes to 404. ;)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9ec15970-7fbb-279f-aba9-4090256c5c7b
* Version Independent ID: 9a97664c-85b4-f1fa-728c-8e56355880f9
* Content: [Designing for section backgrounds using semantic slots](https://docs.microsoft.com/en-us/sharepoint/dev/design/semantic_slots)
* Content Source: [docs/design/semantic_slots.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/design/semantic_slots.md)
* Product: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

### PR DESCRIPTION
**Pull request by omarelanis**
Update broken link for "Fabric semantic colors documentation" to point to Fluent UI theme slot documentation

https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/theme-slots

## Category

- [X] Content fix
- [ ] New article

## Related issues

- fixes #6954
- mentioned in #6954

## What's in this Pull Request?

Fix broken link from FluentUI Github repo to point instead to https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/theme-slots

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[dd8b7c3](https://github.com/omarelanis/sp-dev-docs/commits/dd8b7c35f2eb3b032173b82e58c8586b1c776eb8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/semantic_slots.md](https://github.com/omarelanis/sp-dev-docs/blob/patch-2/docs/design/semantic_slots.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/design/semantic_slots?branch=pr-en-us-6984) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C4%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105041824076554-6984%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=H72jV42svN%2ByaB7%2BTjBzb0COi9KamKc%2BPC1UHaLPz6I%3D&st=2021-05-04T18%3A20%3A17Z&se=2021-06-04T18%3A25%3A17Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/design/semantic_slots.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ In the table below, you can see all eight palette colors defined for the bodyTe
 
 ## Identifying semantic slots in your designs
 
-Semantic slots should be assigned based on the function of a page element. The name of a semantic slot can quickly tell you how it’s meant to be used. You can find all existing semantic slots and their use case examples in the [Fabric semantic colors documentation](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/ISemanticColors.ts).
+Semantic slots should be assigned based on the function of a page element. The name of a semantic slot can quickly tell you how it’s meant to be used. You can find all existing semantic slots and their use case examples in the [Fabric semantic colors documentation](https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/theme-slots).
```
Comment (andrewconnell) [inline docs/design/semantic_slots.md:67]: Please remove the locale from the URL... MSDOCS adds this dynamically based on the user's user agent.

**Review by andrewconnell — CHANGES_REQUESTED** [review]
Please remove the locale from the URL

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[ed901c7](https://github.com/omarelanis/sp-dev-docs/commits/ed901c77ecad4742b16f632ea40a1bd91863daa9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/semantic_slots.md](https://github.com/omarelanis/sp-dev-docs/blob/patch-2/docs/design/semantic_slots.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/design/semantic_slots?branch=pr-en-us-6984) | [Details](#user-content-9b4c957a02678e16dcd221c8c130d087-9a1136a2)

<a id="9b4c957a02678e16dcd221c8c130d087-9a1136a2"></a>
### [docs/design/semantic_slots.md](https://github.com/omarelanis/sp-dev-docs/blob/patch-2/docs/design/semantic_slots.md)
 - **Line 10, Column 607**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/sharepoint/dev/design/design-a-web-part' will be broken in isolated environments. Replace with a relative link.`````
 - **Line 10, Column 803**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/sharepoint/dev/design/themes-colors' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105181103579655-6984%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=30ElpLmNtvUbjMTZqGjyBgro%2FVQmP%2BYRD99LqUqdNX4%3D&st=2021-05-18T10%3A59%3A39Z&se=2021-06-18T11%3A04%3A39Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[906cfb7](https://github.com/omarelanis/sp-dev-docs/commits/906cfb730d19a0e8d42f8c21d7f455e66ecd4b7f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/semantic_slots.md](https://github.com/omarelanis/sp-dev-docs/blob/patch-2/docs/design/semantic_slots.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/design/semantic_slots?branch=pr-en-us-6984) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105181107150506-6984%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=scSnxIxbz0RyaEbbMA885g8ZQNX28cCOPm0wfArJDUk%3D&st=2021-05-18T11%3A03%3A09Z&se=2021-06-18T11%3A08%3A09Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**